### DISCUSSION TITLE
How can I set max_epochs?

### DISCUSSION THREAD

**Discussion by luoqan** [open-mmlab/mmengine#1197] [Q&A] [answered]
I am training FasterRCNN on my self-built dataset, and I already know that max_epochs are mentioned in _train_cfg = dict(type='EpochBasedTrainLoop', max_epochs=12, val_interval=1)_. But I do have a question that how can I set a best max_epochs number, because I didn't find where there is a function like _early_stop_(If loss don't decrease in several epochs, then training will stop aotomatically.)
I am a real rookie here, looking forward for your help!

**Comment by zhouzaida** [comment, marked as answer]
Hi, EarlyStopping feature is supported in EarlyStoppingHook. More details at https://mmengine.readthedocs.io/en/latest/api/generated/mmengine.hooks.EarlyStoppingHook.html#mmengine.hooks.EarlyStoppingHook
